packages/collector/test/tracing/messaging/kafka-avro/test.js

```diff
@@ -46,7 +45,7 @@ let mochaSuiteFn;
  * Installing kafka-avro via optionalDependencies throws an error,
  * see https://github.com/instana/nodejs/pull/486#discussion_r818509109
  */
-const kafkaAvroAllowedVersions = satisfies(process.versions.node, '14');
+const kafkaAvroAllowedVersions = semver.lt(process.versions.node, '16.0.0');
```

Comment:
_question:_ Prevously, these tests were only executed on one Node.js version (14), now they are executed on 10, 12 and 14. Is that what we want?

Reply:
Discussed offline, that's fine.